docs/book/component-guide/deployers/README.md

```diff
@@ -25,10 +25,11 @@ Use deployers when you need request-response patterns, and orchestrators for sch
 
 ### Deployer Flavors
 
-ZenML provides deployer implementations for different deployment environments:
+Out of the box, ZenML comes with a `local` deployer already part of the default stack that deploys pipelines on your local machine in the form of background processes. Additional Deployers are provided by integrations:
```

Comment:
Should we mention that it's part of the default stack in the local.md file above, then?